cmd/argocd/commands/admin/backup.go

```diff
@@ -472,3 +512,18 @@ func updateTracking(bak, live *unstructured.Unstructured) {
 		}
 	}
 }
+
+// skipResource checks if a resource should be skipped based on its labels.
+func skipResource(bak *unstructured.Unstructured, skipResourcesWithLabels []string) bool {
```

Comment:
skipResourcesWithLabels is always the same. You can convert it to a k/v map once instead of for each object.